App.Stocks/Services/CompaniesManager.cs

```diff
@@ -0,0 +1,75 @@
+﻿using App.Configuration;
+using App.Models.Stocks;
+using App.Repositories.Stocks;
+using App.Stocks.View;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace App.Stocks.Services
+{
+	public interface ICompanyManager
+	{
+		IEnumerable<CompanyView> GetCompaniesWithActiveStocks();
```

Comment:
BAL shouldnt manage Dto classes. All mapping logic and View classes should be moved to controller